core/src/main/java/com/novoda/noplayer/PlayerSubtitleTrack.java

```diff
@@ -0,0 +1,107 @@
+package com.novoda.noplayer;
+
+public class PlayerSubtitleTrack {
```

Comment:
This is a public class we give to our clients, I think a lot of this data might never exist in MediaPlayer if we ever implement it. We should probably restrict ourselves to the minimum for now or check what is available. It's easier to add things to a public API than to remove it.